[16:28:37 18-12-2017] (Debug) main.py::<module>.1036: Server Reload...
[16:29:05 18-12-2017] (Debug) main.py::<module>.1037: Server Reload...
[16:29:18 18-12-2017] (Debug) main.py::<module>.1037: Server Reload...
[16:29:21 18-12-2017] (Debug) main.py::<module>.1037: Server Reload...
[16:31:26 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:31:27 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:31:31 18-12-2017] (Success) main.py::index.94: Lade 2 Posts
[16:31:35 18-12-2017] (Success) main.py::index.94: Lade 2 Posts
[16:31:38 18-12-2017] (Success) main.py::index.94: Lade 2 Posts
[16:31:54 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:32:09 18-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): awd@awd.de
[16:32:21 18-12-2017] (Error) main.py::login.140: Gescheiterter Login (Übereinstimmung): marc-friedrich94@web.de
[16:32:41 18-12-2017] (Success) main.py::login.162: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[16:38:20 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:38:33 18-12-2017] (Success) main.py::login.162: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[16:38:40 18-12-2017] (Success) main.py::logout.185: Benutzer wurde ausgeloggt: marc-friedrich94@web.de
[16:39:18 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:39:53 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:39:55 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:40:06 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:40:09 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:40:13 18-12-2017] (Debug) main.py::<module>.1038: Server Reload...
[16:40:45 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:40:47 18-12-2017] (Debug) main.py::<module>.1038: Server Reload...
[16:41:15 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:41:32 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:41:42 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:41:44 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:42:21 18-12-2017] (Debug) main.py::<module>.1038: Server Reload...
[16:42:45 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:42:58 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:44:30 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:44:59 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:45:05 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:45:07 18-12-2017] (Debug) main.py::<module>.1043: Server Reload...
[16:45:22 18-12-2017] (Debug) main.py::<module>.1044: Server Reload...
[16:46:02 18-12-2017] (Debug) main.py::<module>.1045: Server Reload...
[16:47:52 18-12-2017] (Debug) main.py::<module>.1046: Server Reload...
[16:48:00 18-12-2017] (Debug) main.py::<module>.1046: Server Reload...
[16:48:03 18-12-2017] (Debug) main.py::<module>.1044: Server Reload...
[16:48:27 18-12-2017] (Debug) main.py::<module>.1045: Server Reload...
[16:48:37 18-12-2017] (Debug) main.py::<module>.1044: Server Reload...
[16:48:59 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:49:38 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:49:54 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:50:07 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:50:12 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:51:09 18-12-2017] (Debug) main.py::<module>.1049: Server Reload...
[16:51:15 18-12-2017] (Debug) main.py::<module>.1057: Server Reload...
[16:51:22 18-12-2017] (Debug) main.py::<module>.1065: Server Reload...
[16:51:29 18-12-2017] (Debug) main.py::<module>.1073: Server Reload...
[16:52:14 18-12-2017] (Debug) main.py::<module>.1081: Server Reload...
[16:53:26 18-12-2017] (Debug) main.py::<module>.1082: Server Reload...
[16:53:46 18-12-2017] (Debug) main.py::<module>.1082: Server Reload...
[16:53:58 18-12-2017] (Debug) main.py::<module>.1083: Server Reload...
[16:54:18 18-12-2017] (Debug) main.py::<module>.1092: Server Reload...
[16:54:38 18-12-2017] (Debug) main.py::<module>.1093: Server Reload...
[16:54:44 18-12-2017] (Debug) main.py::<module>.1094: Server Reload...
[16:54:53 18-12-2017] (Debug) main.py::<module>.1095: Server Reload...
[16:55:03 18-12-2017] (Debug) main.py::<module>.1096: Server Reload...
[16:55:10 18-12-2017] (Debug) main.py::<module>.1097: Server Reload...
[16:56:39 18-12-2017] (Debug) main.py::<module>.1098: Server Reload...
[16:57:49 18-12-2017] (Debug) main.py::<module>.1099: Server Reload...
[16:57:51 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[16:58:16 18-12-2017] (Debug) main.py::<module>.1102: Server Reload...
[16:58:46 18-12-2017] (Success) main.py::login.162: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[16:59:07 18-12-2017] (Success) main.py::post_message.368: Neue Nachricht von Benuzter marc-friedrich94@web.de geposted
[16:59:53 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[17:00:08 18-12-2017] (Debug) main.py::<module>.1100: Server Reload...
[17:00:23 18-12-2017] (Debug) main.py::<module>.1100: Server Reload...
[17:00:53 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[17:00:57 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[17:01:25 18-12-2017] (Debug) main.py::<module>.1102: Server Reload...
[17:01:48 18-12-2017] (Success) main.py::vote.407: Benutzer marc-friedrich94@web.de hat bereits für Post 5 abgestimmt
[17:02:43 18-12-2017] (Debug) main.py::<module>.1103: Server Reload...
[17:03:23 18-12-2017] (Debug) main.py::<module>.1104: Server Reload...
[17:05:42 18-12-2017] (Debug) main.py::<module>.1105: Server Reload...
[17:06:26 18-12-2017] (Debug) main.py::<module>.1106: Server Reload...
[17:07:07 18-12-2017] (Debug) main.py::<module>.1107: Server Reload...
[17:07:50 18-12-2017] (Debug) main.py::<module>.1109: Server Reload...
[17:10:37 18-12-2017] (Debug) main.py::<module>.1110: Server Reload...
[17:10:46 18-12-2017] (Debug) main.py::<module>.1111: Server Reload...
[17:11:25 18-12-2017] (Debug) main.py::<module>.1112: Server Reload...
[17:11:38 18-12-2017] (Debug) main.py::<module>.1113: Server Reload...
[17:46:56 18-12-2017] (Debug) main.py::<module>.1114: Server Reload...
[17:47:15 18-12-2017] (Debug) main.py::<module>.1115: Server Reload...
[17:47:42 18-12-2017] (Debug) main.py::<module>.1116: Server Reload...
[17:50:56 18-12-2017] (Debug) main.py::<module>.1116: Server Reload...
[17:51:34 18-12-2017] (Debug) main.py::<module>.1117: Server Reload...
[17:51:36 18-12-2017] (Debug) main.py::<module>.1117: Server Reload...
[17:52:51 18-12-2017] (Debug) main.py::<module>.1118: Server Reload...
[17:53:00 18-12-2017] (Debug) main.py::<module>.1116: Server Reload...
[17:53:38 18-12-2017] (Debug) main.py::<module>.1117: Server Reload...
[17:54:02 18-12-2017] (Debug) main.py::<module>.1118: Server Reload...
[17:54:37 18-12-2017] (Debug) main.py::<module>.1119: Server Reload...
[17:55:08 18-12-2017] (Debug) main.py::<module>.1120: Server Reload...
[17:56:19 18-12-2017] (Debug) main.py::<module>.1121: Server Reload...
[17:56:53 18-12-2017] (Debug) main.py::<module>.1123: Server Reload...
[17:57:13 18-12-2017] (Debug) main.py::<module>.1124: Server Reload...
[17:57:34 18-12-2017] (Debug) main.py::<module>.1126: Server Reload...
[17:58:10 18-12-2017] (Debug) main.py::<module>.1127: Server Reload...
[17:58:15 18-12-2017] (Debug) main.py::<module>.1128: Server Reload...
[17:58:28 18-12-2017] (Debug) main.py::<module>.1127: Server Reload...
[17:58:40 18-12-2017] (Debug) main.py::<module>.1128: Server Reload...
[17:58:45 18-12-2017] (Debug) main.py::<module>.1128: Server Reload...
[17:59:04 18-12-2017] (Debug) main.py::<module>.1130: Server Reload...
[17:59:13 18-12-2017] (Debug) main.py::<module>.1131: Server Reload...
[18:03:08 18-12-2017] (Debug) main.py::<module>.1132: Server Reload...
[18:04:15 18-12-2017] (Debug) main.py::<module>.1133: Server Reload...
[18:04:29 18-12-2017] (Debug) main.py::<module>.1134: Server Reload...
[18:04:31 18-12-2017] (Debug) main.py::<module>.1134: Server Reload...
[18:05:02 18-12-2017] (Debug) main.py::<module>.1135: Server Reload...
[18:05:23 18-12-2017] (Debug) main.py::<module>.1136: Server Reload...
[18:05:59 18-12-2017] (Debug) main.py::<module>.1137: Server Reload...
[18:06:07 18-12-2017] (Debug) main.py::<module>.1137: Server Reload...
[18:06:28 18-12-2017] (Debug) main.py::<module>.1138: Server Reload...
[18:06:38 18-12-2017] (Debug) main.py::<module>.1137: Server Reload...
[18:07:05 18-12-2017] (Debug) main.py::<module>.1138: Server Reload...
[18:07:09 18-12-2017] (Debug) main.py::<module>.1138: Server Reload...
[18:13:58 18-12-2017] (Debug) main.py::<module>.1139: Server Reload...
[18:14:19 18-12-2017] (Debug) main.py::<module>.1141: Server Reload...
[18:14:56 18-12-2017] (Debug) main.py::<module>.1142: Server Reload...
[18:15:12 18-12-2017] (Debug) main.py::<module>.1143: Server Reload...
[18:15:44 18-12-2017] (Debug) main.py::<module>.1143: Server Reload...
[18:16:12 18-12-2017] (Debug) main.py::<module>.1145: Server Reload...
[18:16:14 18-12-2017] (Debug) main.py::<module>.1145: Server Reload...
[18:16:18 18-12-2017] (Debug) main.py::<module>.1145: Server Reload...
[18:18:28 18-12-2017] (Debug) main.py::<module>.1146: Server Reload...
[18:18:29 18-12-2017] (Debug) main.py::<module>.1146: Server Reload...
[18:19:59 18-12-2017] (Debug) main.py::<module>.1149: Server Reload...
[18:20:41 18-12-2017] (Debug) main.py::<module>.1151: Server Reload...
[18:21:45 18-12-2017] (Debug) main.py::<module>.1152: Server Reload...
[18:22:18 18-12-2017] (Debug) main.py::<module>.1153: Server Reload...
[18:22:48 18-12-2017] (Debug) main.py::<module>.1154: Server Reload...
[18:23:29 18-12-2017] (Debug) main.py::<module>.1156: Server Reload...
[18:24:08 18-12-2017] (Debug) main.py::<module>.1157: Server Reload...
[18:25:01 18-12-2017] (Debug) main.py::<module>.1159: Server Reload...
[18:25:40 18-12-2017] (Debug) main.py::<module>.1160: Server Reload...
[18:26:25 18-12-2017] (Debug) main.py::<module>.1163: Server Reload...
[18:26:47 18-12-2017] (Debug) main.py::<module>.1158: Server Reload...
[18:26:54 18-12-2017] (Debug) main.py::<module>.1156: Server Reload...
[18:27:30 18-12-2017] (Debug) main.py::<module>.1154: Server Reload...
[18:27:40 18-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[18:28:22 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: marc-friedrich94@web.de
[18:29:36 18-12-2017] (Success) main.py::handle_password_reset.942: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc-friedrich94@web.de
[18:30:18 18-12-2017] (Error) main.py::confirm_password_reset.980: Übermitteltes Token stimmt nicht mit registriertem Token überein. Mögliche Attacke (SQLi) erkannt.
[18:30:34 18-12-2017] (Success) main.py::set_new_password.1029: Passwort für Benutzer 9 wurde geändert.
[18:30:35 18-12-2017] (Debug) main.py::set_new_password.1033: Logge Benutzer 9 aus, um sich mit den geänderten Credentials erneut auszuloggen.
[18:30:53 18-12-2017] (Success) main.py::handle_password_reset.942: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc-friedrich94@web.de
[18:31:19 18-12-2017] (Success) main.py::set_new_password.1029: Passwort für Benutzer 9 wurde geändert.
[18:31:20 18-12-2017] (Debug) main.py::set_new_password.1033: Logge Benutzer 9 aus, um sich mit den geänderten Credentials erneut auszuloggen.
[18:31:29 18-12-2017] (Error) main.py::login.139: Gescheiterter Login (Übereinstimmung): marc-friedrich94@web.de
[18:31:39 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[18:32:35 18-12-2017] (Success) main.py::change_password_handler.714: Bestätigungsmail für Passwortänderung wurde an marc-friedrich94@web.de gesendet.
[18:33:03 18-12-2017] (Success) main.py::confirm_password_change.767: Benutzer 9 hat Passwort geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[18:33:03 18-12-2017] (Debug) main.py::confirm_password_change.770: Logge User aus, um sich mit den neuen Credentials anzumelden.
[18:33:24 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[18:33:54 18-12-2017] (Success) main.py::change_email_handler.580: Bestätigungsmail für E-Mail Änderung wurde an marc-friedrich94@web.de gesendet.
[18:36:13 18-12-2017] (Success) main.py::change_email_handler.580: Bestätigungsmail für E-Mail Änderung wurde an marc-friedrich94@web.de gesendet.
[18:36:53 18-12-2017] (Success) main.py::confirm_email_change.632: Benutzer 9 hat seine E-Mail geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[18:36:53 18-12-2017] (Debug) main.py::confirm_email_change.635: Logge User aus, um sich mit den neuen Credentials anzumelden.
[18:37:04 18-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): marc-friedrich94@web.de
[18:37:13 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:37:20 18-12-2017] (Success) main.py::admin_dashboard.280: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[18:37:28 18-12-2017] (Success) main.py::admin_dashboard.280: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[18:37:31 18-12-2017] (Success) main.py::admin_confirm.891: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 4
[18:37:35 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:38:03 18-12-2017] (Success) main.py::login.160: 
[18:38:14 18-12-2017] (Error) main.py::admin_dashboard.265: Unbefugter Benutzer 'Ingo.piller@hans-maier.de versucht auf das Admin Dashboard zuzugreifen. Verweigere Zugriff.
[18:38:38 18-12-2017] (Error) main.py::delete_user.802: Unbefugter Benutzer Ingo.piller@hans-maier.de wollte User löschen. Mögliche Attacke erkannt.
[18:39:00 18-12-2017] (Success) main.py::post_message.366: Neue Nachricht von Benuzter Ingo.piller@hans-maier.de geposted
[18:39:11 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:39:18 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 3 von Ingo.piller@hans-maier.de wurde registriert.
[18:39:30 18-12-2017] (Success) main.py::vote.407: Benutzer Ingo.piller@hans-maier.de hat bereits für Post 3 abgestimmt
[18:39:41 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:39:44 18-12-2017] (Error) main.py::finish_vote.455: Vote konnte nicht abgeschlossen werden (keine Übereinstimmung der Token)
[18:39:55 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:40:02 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 4 von Ingo.piller@hans-maier.de wurde registriert.
[18:40:15 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:40:26 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 6 von Ingo.piller@hans-maier.de wurde registriert.
[18:40:31 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: Ingo.piller@hans-maier.de
[18:40:41 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:40:48 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:40:55 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 6 von marc.friedrich94@web.de wurde registriert.
[18:41:07 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:41:14 18-12-2017] (Success) main.py::login.160: 
[18:41:22 18-12-2017] (Success) main.py::vote.407: Benutzer Ingo.piller@hans-maier.de hat bereits für Post 6 abgestimmt
[18:43:33 18-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[10:03:50 31-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[10:03:51 31-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[10:04:16 31-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): marc-friedrich94@web.de
[10:04:24 31-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): marc-friedrich94@web.de
[10:04:35 31-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): marc-friedrich94@web.de
[10:05:04 31-12-2017] (Error) main.py::login.139: Gescheiterter Login (Übereinstimmung): marc.friedrich94@web.de
[10:05:14 31-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[10:05:19 31-12-2017] (Success) main.py::admin_dashboard.280: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[10:05:38 31-12-2017] (Success) main.py::vote.407: Benutzer marc.friedrich94@web.de hat bereits für Post 6 abgestimmt
[10:05:44 31-12-2017] (Success) main.py::vote.407: Benutzer marc.friedrich94@web.de hat bereits für Post 5 abgestimmt
[10:05:50 31-12-2017] (Success) main.py::vote.407: Benutzer marc.friedrich94@web.de hat bereits für Post 4 abgestimmt
[10:05:56 31-12-2017] (Success) main.py::vote.407: Benutzer marc.friedrich94@web.de hat bereits für Post 3 abgestimmt
[10:19:08 31-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[10:19:23 31-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[10:19:56 31-12-2017] (Debug) main.py::<module>.1152: Server Reload...
[10:20:27 31-12-2017] (Debug) main.py::<module>.1152: Server Reload...
[10:20:46 31-12-2017] (Debug) main.py::<module>.1152: Server Reload...
[10:21:44 31-12-2017] (Debug) main.py::<module>.1152: Server Reload...
[10:22:39 31-12-2017] (Debug) main.py::<module>.1156: Server Reload...
[10:23:46 31-12-2017] (Debug) main.py::<module>.1161: Server Reload...
[10:24:49 31-12-2017] (Debug) main.py::<module>.1162: Server Reload...
[10:25:52 31-12-2017] (Debug) main.py::<module>.1162: Server Reload...
[10:26:01 31-12-2017] (Debug) main.py::<module>.1162: Server Reload...
[10:26:20 31-12-2017] (Debug) main.py::<module>.1162: Server Reload...
[10:26:22 31-12-2017] (Debug) main.py::<module>.1163: Server Reload...
[10:26:25 31-12-2017] (Debug) main.py::<module>.1163: Server Reload...
[10:28:46 31-12-2017] (Debug) main.py::<module>.1164: Server Reload...
[10:29:22 31-12-2017] (Debug) main.py::<module>.1164: Server Reload...
[10:29:37 31-12-2017] (Debug) main.py::<module>.1166: Server Reload...
[10:30:29 31-12-2017] (Debug) main.py::<module>.1166: Server Reload...
[10:30:39 31-12-2017] (Debug) main.py::<module>.1166: Server Reload...
[10:30:48 31-12-2017] (Debug) main.py::<module>.1166: Server Reload...
[10:30:55 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:31:08 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:31:25 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:31:36 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:32:21 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:32:26 31-12-2017] (Debug) main.py::<module>.1166: Server Reload...
[10:33:31 31-12-2017] (Debug) main.py::<module>.1170: Server Reload...
[10:36:07 31-12-2017] (Debug) main.py::<module>.1168: Server Reload...
[10:37:08 31-12-2017] (Debug) main.py::<module>.1168: Server Reload...
[10:37:44 31-12-2017] (Debug) main.py::<module>.1164: Server Reload...
[10:37:55 31-12-2017] (Debug) main.py::<module>.1164: Server Reload...
[10:38:07 31-12-2017] (Debug) main.py::<module>.1164: Server Reload...
[10:38:18 31-12-2017] (Debug) main.py::<module>.1164: Server Reload...
[10:39:14 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:39:15 31-12-2017] (Debug) main.py::<module>.1165: Server Reload...
[10:39:54 31-12-2017] (Debug) main.py::<module>.1167: Server Reload...
[10:39:58 31-12-2017] (Debug) main.py::<module>.1167: Server Reload...
[10:40:15 31-12-2017] (Debug) main.py::<module>.1168: Server Reload...
[10:41:21 31-12-2017] (Debug) main.py::<module>.1169: Server Reload...
[10:42:05 31-12-2017] (Debug) main.py::<module>.1169: Server Reload...
[10:42:38 31-12-2017] (Debug) main.py::<module>.1171: Server Reload...
[10:43:11 31-12-2017] (Debug) main.py::<module>.1172: Server Reload...
[10:44:43 31-12-2017] (Debug) main.py::<module>.1171: Server Reload...
[10:44:49 31-12-2017] (Debug) main.py::<module>.1172: Server Reload...
[10:45:06 31-12-2017] (Debug) main.py::<module>.1172: Server Reload...
[10:45:28 31-12-2017] (Debug) main.py::<module>.1172: Server Reload...
[10:47:25 31-12-2017] (Debug) main.py::<module>.1173: Server Reload...
[10:48:58 31-12-2017] (Debug) main.py::<module>.1179: Server Reload...
[10:49:54 31-12-2017] (Debug) main.py::<module>.1181: Server Reload...
[10:53:00 31-12-2017] (Debug) main.py::<module>.1182: Server Reload...
[10:53:19 31-12-2017] (Debug) main.py::<module>.1181: Server Reload...
[10:54:01 31-12-2017] (Debug) main.py::<module>.1181: Server Reload...
[10:54:17 31-12-2017] (Debug) main.py::<module>.1181: Server Reload...
[10:54:46 31-12-2017] (Debug) main.py::<module>.1181: Server Reload...
[10:55:12 31-12-2017] (Debug) main.py::<module>.1184: Server Reload...
[10:55:36 31-12-2017] (Debug) main.py::<module>.1188: Server Reload...
[10:56:00 31-12-2017] (Debug) main.py::<module>.1188: Server Reload...
[11:00:47 31-12-2017] (Success) main.py::post_message.404: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:01:12 31-12-2017] (Success) main.py::post_message.404: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:01:55 31-12-2017] (Debug) main.py::<module>.1189: Server Reload...
[11:03:29 31-12-2017] (Debug) main.py::<module>.1190: Server Reload...
[11:04:12 31-12-2017] (Success) main.py::post_message.406: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:04:38 31-12-2017] (Success) main.py::post_message.406: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:04:59 31-12-2017] (Success) main.py::post_message.406: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:05:20 31-12-2017] (Error) main.py::post_message.376: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[11:05:30 31-12-2017] (Success) main.py::login.173: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[11:05:47 31-12-2017] (Success) main.py::post_message.406: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:06:15 31-12-2017] (Debug) main.py::<module>.1190: Server Reload...
[11:06:41 31-12-2017] (Debug) main.py::<module>.1192: Server Reload...
[11:11:14 31-12-2017] (Debug) main.py::<module>.1191: Server Reload...
[11:13:32 31-12-2017] (Debug) main.py::<module>.1191: Server Reload...
[11:15:43 31-12-2017] (Debug) main.py::<module>.1191: Server Reload...
[11:15:47 31-12-2017] (Debug) main.py::<module>.1193: Server Reload...
[11:26:17 31-12-2017] (Debug) main.py::<module>.1207: Server Reload...
[11:26:18 31-12-2017] (Debug) main.py::<module>.1207: Server Reload...
[11:29:18 31-12-2017] (Debug) main.py::<module>.1210: Server Reload...
[11:29:53 31-12-2017] (Debug) main.py::<module>.1208: Server Reload...
[11:29:56 31-12-2017] (Debug) main.py::<module>.1207: Server Reload...
[11:30:47 31-12-2017] (Debug) main.py::<module>.1207: Server Reload...
[11:31:01 31-12-2017] (Debug) main.py::<module>.1209: Server Reload...
[11:31:58 31-12-2017] (Debug) main.py::<module>.1213: Server Reload...
[11:34:58 31-12-2017] (Debug) main.py::<module>.1217: Server Reload...
[11:35:47 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:35:50 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:37:36 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:37:38 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:38:03 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:38:37 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:39:00 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:41:54 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:41:57 31-12-2017] (Debug) main.py::<module>.1219: Server Reload...
[11:42:19 31-12-2017] (Debug) main.py::<module>.1233: Server Reload...
[11:42:45 31-12-2017] (Debug) main.py::<module>.1234: Server Reload...
[11:43:11 31-12-2017] (Debug) main.py::<module>.1241: Server Reload...
[11:43:20 31-12-2017] (Debug) main.py::<module>.1241: Server Reload...
[11:43:24 31-12-2017] (Debug) main.py::<module>.1241: Server Reload...
[11:43:32 31-12-2017] (Debug) main.py::<module>.1241: Server Reload...
[11:44:09 31-12-2017] (Debug) main.py::<module>.1245: Server Reload...
[11:44:26 31-12-2017] (Debug) main.py::<module>.1249: Server Reload...
[14:01:47 31-12-2017] (Debug) main.py::<module>.1249: Server Reload...
[14:03:55 31-12-2017] (Debug) main.py::<module>.1254: Server Reload...
[14:05:09 31-12-2017] (Debug) main.py::<module>.1254: Server Reload...
[14:06:09 31-12-2017] (Debug) main.py::<module>.1252: Server Reload...
[14:06:22 31-12-2017] (Debug) main.py::<module>.1252: Server Reload...
[14:06:24 31-12-2017] (Debug) main.py::<module>.1252: Server Reload...
[14:07:55 31-12-2017] (Debug) main.py::<module>.1255: Server Reload...
[14:09:11 31-12-2017] (Debug) main.py::<module>.1261: Server Reload...
[14:09:34 31-12-2017] (Debug) main.py::<module>.1262: Server Reload...
[14:10:23 31-12-2017] (Debug) main.py::<module>.1265: Server Reload...
[14:10:36 31-12-2017] (Debug) main.py::<module>.1265: Server Reload...
[14:11:08 31-12-2017] (Debug) main.py::<module>.1266: Server Reload...
[14:11:34 31-12-2017] (Debug) main.py::<module>.1264: Server Reload...
[14:12:51 31-12-2017] (Debug) main.py::<module>.1264: Server Reload...
[14:13:45 31-12-2017] (Debug) main.py::<module>.1265: Server Reload...
[14:14:49 31-12-2017] (Debug) main.py::<module>.1265: Server Reload...
[14:15:12 31-12-2017] (Success) main.py::logout.199: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[14:15:14 31-12-2017] (Error) main.py::confirm_password_reset.1081: Konnte die übergebenen URL-Parameter nicht auswerten.
[14:16:21 31-12-2017] (Success) main.py::login.173: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[14:16:30 31-12-2017] (Success) main.py::vote.466: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[14:16:38 31-12-2017] (Success) main.py::finish_vote.550: Downvote für 12 von marc.friedrich94@web.de wurde registriert.
[14:16:46 31-12-2017] (Success) main.py::admin_dashboard.302: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[14:17:44 31-12-2017] (Debug) main.py::<module>.1265: Server Reload...
[14:18:22 31-12-2017] (Debug) main.py::<module>.1267: Server Reload...
[14:19:13 31-12-2017] (Debug) main.py::<module>.1269: Server Reload...
[14:19:31 31-12-2017] (Debug) main.py::<module>.1273: Server Reload...
[14:20:53 31-12-2017] (Debug) main.py::<module>.1276: Server Reload...
[14:21:09 31-12-2017] (Debug) main.py::<module>.1276: Server Reload...
[14:25:53 31-12-2017] (Debug) main.py::<module>.1276: Server Reload...
[14:25:56 31-12-2017] (Debug) main.py::<module>.1276: Server Reload...
[14:28:18 31-12-2017] (Debug) main.py::<module>.1277: Server Reload...
[14:28:20 31-12-2017] (Debug) main.py::<module>.1276: Server Reload...
[14:32:28 31-12-2017] (Debug) main.py::<module>.1280: Server Reload...
[14:32:29 31-12-2017] (Debug) main.py::<module>.1280: Server Reload...
[14:33:49 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:33:51 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:33:57 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:33:58 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:27 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:30 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:43 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:44 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:47 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:49 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:34:58 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:35:00 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:35:32 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:35:33 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:02 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:03 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:09 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:10 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:12 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:26 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:28 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:45 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:36:47 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:37:00 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:37:02 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:37:04 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:37:06 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:37:16 31-12-2017] (Debug) main.py::<module>.1283: Server Reload...
[14:37:18 31-12-2017] (Debug) main.py::<module>.1283: Server Reload...
[14:38:25 31-12-2017] (Debug) main.py::<module>.1278: Server Reload...
[14:38:27 31-12-2017] (Debug) main.py::<module>.1278: Server Reload...
[14:38:33 31-12-2017] (Debug) main.py::<module>.1279: Server Reload...
[14:38:35 31-12-2017] (Debug) main.py::<module>.1279: Server Reload...
[14:38:40 31-12-2017] (Debug) main.py::<module>.1279: Server Reload...
[14:38:41 31-12-2017] (Debug) main.py::<module>.1279: Server Reload...
[14:38:48 31-12-2017] (Debug) main.py::<module>.1278: Server Reload...
[14:38:49 31-12-2017] (Debug) main.py::<module>.1278: Server Reload...
[14:39:22 31-12-2017] (Debug) main.py::<module>.1282: Server Reload...
[14:39:24 31-12-2017] (Debug) main.py::<module>.1282: Server Reload...
[14:39:42 31-12-2017] (Debug) main.py::<module>.1287: Server Reload...
[14:39:44 31-12-2017] (Debug) main.py::<module>.1287: Server Reload...
[14:40:18 31-12-2017] (Debug) main.py::<module>.1289: Server Reload...
[14:40:20 31-12-2017] (Debug) main.py::<module>.1289: Server Reload...
[14:40:32 31-12-2017] (Debug) main.py::<module>.1290: Server Reload...
[14:40:33 31-12-2017] (Debug) main.py::<module>.1290: Server Reload...
[14:40:57 31-12-2017] (Debug) main.py::<module>.1293: Server Reload...
[14:40:58 31-12-2017] (Debug) main.py::<module>.1293: Server Reload...
[14:42:49 31-12-2017] (Debug) main.py::<module>.1294: Server Reload...
[14:42:51 31-12-2017] (Debug) main.py::<module>.1294: Server Reload...
[14:42:53 31-12-2017] (Debug) main.py::<module>.1291: Server Reload...
[14:43:10 31-12-2017] (Debug) main.py::<module>.1290: Server Reload...
[14:43:11 31-12-2017] (Debug) main.py::<module>.1290: Server Reload...
[14:43:30 31-12-2017] (Debug) main.py::<module>.1291: Server Reload...
[14:43:31 31-12-2017] (Debug) main.py::<module>.1291: Server Reload...
[14:43:36 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:43:38 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:43:47 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:43:48 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:16 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:18 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:19 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:20 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:32 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:33 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:35 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:37 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:42 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:44:44 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:45:10 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:45:12 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:45:19 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:45:21 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:45:32 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:45:34 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:46:02 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:46:04 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:46:29 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:46:30 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:46:51 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:46:53 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:03 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:05 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:05 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:17 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:18 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:48 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:47:51 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:48:04 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:48:06 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:48:58 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[14:49:00 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:10:01 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:10:02 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:11:32 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:11:33 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:12:19 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:12:20 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:12:26 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:12:27 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:12:41 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:12:42 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:13:02 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:13:03 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:13:18 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:13:19 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:15:38 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:15:39 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:16:47 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:16:49 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:04 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:05 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:20 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:21 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:26 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:28 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:46 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:17:47 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:18:32 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:18:33 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:14 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:16 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:18 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:20 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:39 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:41 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:47 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:19:48 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:20:15 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:20:17 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:20:52 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:20:54 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:20:58 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:20:59 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:21:05 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:21:07 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:23:13 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:23:15 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:23:38 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[15:23:39 31-12-2017] (Debug) main.py::<module>.1288: Server Reload...
[17:36:33 02-01-2018] (Debug) main.py::<module>.1303: Server Reload...
[17:36:34 02-01-2018] (Debug) main.py::<module>.1303: Server Reload...
[17:39:21 02-01-2018] (Success) main.py::login.179: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[17:39:54 02-01-2018] (Success) main.py::logout.205: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[17:40:48 02-01-2018] (Success) main.py::login.179: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[17:42:10 02-01-2018] (Error) main.py::check_if_valid_session.1290: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[17:43:11 02-01-2018] (Error) main.py::check_if_valid_session.1290: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[17:43:58 02-01-2018] (Debug) main.py::<module>.1303: Server Reload...
[17:44:03 02-01-2018] (Debug) main.py::<module>.1303: Server Reload...
[17:44:19 02-01-2018] (Error) main.py::check_if_valid_session.1290: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[17:44:50 02-01-2018] (Success) main.py::logout.205: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[17:45:02 02-01-2018] (Success) main.py::login.179: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[17:46:12 02-01-2018] (Error) main.py::check_if_valid_session.1290: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[17:48:11 02-01-2018] (Debug) main.py::<module>.1303: Server Reload...
[17:48:25 02-01-2018] (Debug) main.py::<module>.1303: Server Reload...
[17:48:33 02-01-2018] (Debug) main.py::<module>.1305: Server Reload...
[17:48:36 02-01-2018] (Debug) main.py::<module>.1306: Server Reload...
[17:48:37 02-01-2018] (Success) main.py::logout.205: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[17:49:18 02-01-2018] (Success) main.py::login.179: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[17:51:22 02-01-2018] (Error) main.py::check_if_valid_session.1292: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[17:52:13 02-01-2018] (Success) main.py::login.179: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:04:15 02-01-2018] (Debug) main.py::<module>.1306: Server Reload...
[18:04:25 02-01-2018] (Debug) main.py::<module>.1297: Server Reload...
[18:07:27 02-01-2018] (Debug) main.py::<module>.1302: Server Reload...
[18:07:40 02-01-2018] (Debug) main.py::<module>.1299: Server Reload...
[18:08:56 02-01-2018] (Debug) main.py::<module>.1295: Server Reload...
[18:09:17 02-01-2018] (Debug) main.py::<module>.1296: Server Reload...
[18:09:30 02-01-2018] (Debug) main.py::<module>.1296: Server Reload...
[18:09:37 02-01-2018] (Debug) main.py::<module>.1292: Server Reload...
[18:09:53 02-01-2018] (Debug) main.py::<module>.1293: Server Reload...
[18:10:44 02-01-2018] (Debug) main.py::<module>.1293: Server Reload...
[18:20:01 02-01-2018] (Debug) main.py::<module>.1289: Server Reload...
[18:20:18 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[18:20:25 02-01-2018] (Debug) main.py::<module>.1281: Server Reload...
[18:20:35 02-01-2018] (Debug) main.py::<module>.1277: Server Reload...
[18:20:47 02-01-2018] (Debug) main.py::<module>.1273: Server Reload...
[18:24:22 02-01-2018] (Debug) main.py::<module>.1273: Server Reload...
[18:24:25 02-01-2018] (Debug) main.py::<module>.1273: Server Reload...
[18:25:00 02-01-2018] (Debug) main.py::<module>.1275: Server Reload...
[18:25:31 02-01-2018] (Debug) main.py::<module>.1277: Server Reload...
[18:26:01 02-01-2018] (Debug) main.py::<module>.1277: Server Reload...
[18:30:46 02-01-2018] (Success) main.py::logout.207: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:38:21 02-01-2018] (Success) main.py::handle_password_reset.1037: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc.friedrich94@web.de
[18:44:31 02-01-2018] (Error) main.py::confirm_password_reset.1078: Übermitteltes Token stimmt nicht mit registriertem Token überein. Mögliche Attacke (SQLi) erkannt.
[18:44:59 02-01-2018] (Success) main.py::set_new_password.1134: Passwort für Benutzer 9 wurde geändert.
[18:45:01 02-01-2018] (Debug) main.py::set_new_password.1138: Logge Benutzer 9 aus, um sich mit den geänderten Credentials erneut auszuloggen.
[18:46:36 02-01-2018] (Debug) main.py::<module>.1277: Server Reload...
[18:47:46 02-01-2018] (Debug) main.py::<module>.1277: Server Reload...
[18:47:55 02-01-2018] (Error) main.py::confirm_password_reset.1072: Kein zugehöriger Passwort Reset Request für Token None gefunden.
[18:48:03 02-01-2018] (Error) main.py::confirm_password_reset.1072: Kein zugehöriger Passwort Reset Request für Token None gefunden.
[18:48:58 02-01-2018] (Success) main.py::handle_password_reset.1037: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc.friedrich94@web.de
[18:49:24 02-01-2018] (Error) main.py::confirm_password_reset.1072: Kein zugehöriger Passwort Reset Request für Token None gefunden.
[18:50:23 02-01-2018] (Error) main.py::set_new_password.1124: Das übermittelte Reset Token stimmt leider nicht mit dem registrierten Token überein.
[18:52:49 02-01-2018] (Debug) main.py::<module>.1278: Server Reload...
[18:52:57 02-01-2018] (Success) main.py::set_new_password.1135: Passwort für Benutzer 9 wurde geändert.
[18:52:58 02-01-2018] (Debug) main.py::set_new_password.1139: Logge Benutzer 9 aus, um sich mit den geänderten Credentials erneut einzuloggen.
[18:53:32 02-01-2018] (Error) main.py::login.159: Gescheiterter Login (Übereinstimmung): hans.meier@hans-maier.de
[18:53:51 02-01-2018] (Success) main.py::login.181: 
[18:53:56 02-01-2018] (Success) main.py::logout.207: Benutzer wurde ausgeloggt: hans.meier@hans-maier.de
[18:55:30 02-01-2018] (Success) main.py::login.181: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:55:50 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:55:59 02-01-2018] (Success) main.py::vote.451: Benutzer marc.friedrich94@web.de hat bereits für Post 4 abgestimmt
[18:56:07 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:56:26 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:56:58 02-01-2018] (Error) main.py::finish_vote.506: Vote konnte nicht abgeschlossen werden (keine Übereinstimmung der Token)
[18:57:10 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:59:40 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[19:00:10 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[19:00:13 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[19:00:29 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[19:00:49 02-01-2018] (Success) main.py::finish_vote.535: Upvote für 6 von marc.friedrich94@web.de wurde registriert.
[19:01:01 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:01:15 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:01:21 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:04:28 02-01-2018] (Success) main.py::vote.451: Benutzer marc.friedrich94@web.de hat bereits für Post 6 abgestimmt
[19:04:33 02-01-2018] (Success) main.py::vote.451: Benutzer marc.friedrich94@web.de hat bereits für Post 6 abgestimmt
[19:04:52 02-01-2018] (Success) main.py::post_message.413: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[19:05:02 02-01-2018] (Success) main.py::vote.468: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[19:05:09 02-01-2018] (Success) main.py::finish_vote.535: Upvote für 13 von marc.friedrich94@web.de wurde registriert.
[19:06:06 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:06:10 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:06:40 02-01-2018] (Success) main.py::vote.451: Benutzer marc.friedrich94@web.de hat bereits für Post 13 abgestimmt
[19:06:48 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:06:51 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:07:01 02-01-2018] (Error) main.py::finish_vote.521: Benutzer marc.friedrich94@web.de hat bereits für den Post abgestimmt.
[19:07:20 02-01-2018] (Success) main.py::admin_dashboard.312: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[19:07:23 02-01-2018] (Success) main.py::admin_confirm.972: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 6
[19:32:00 02-01-2018] (Debug) main.py::<module>.1278: Server Reload...
[19:32:54 02-01-2018] (Debug) main.py::<module>.1278: Server Reload...
[19:33:34 02-01-2018] (Debug) main.py::<module>.1278: Server Reload...
[19:44:09 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:44:27 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:45:28 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:46:15 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:46:43 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:47:24 02-01-2018] (Success) main.py::logout.205: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[19:47:31 02-01-2018] (Success) main.py::logout.205: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[19:48:01 02-01-2018] (Success) main.py::send_verification_email.1225: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[19:48:01 02-01-2018] (Success) main.py::post_user.272: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[19:48:26 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (hhrgswcnm8jy71he014f2.xx65xbwzbxa3vdoqx7stgkxxoibva)
[19:48:41 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden ()
[19:48:44 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (awd)
[19:48:51 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (hhrgswcnm8jy71hex7stgkxxoibva)
[19:49:00 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (12)
[19:49:25 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (hhrgswcnm8jy71he014f2xx65xbwzbxa3vdswaoqx7stgkxxoibva)
[19:49:34 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (hhrgswcnm8jy71he014f2xx65xbwzbxa3vdoqx7stgkxxoibva)
[19:51:12 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:51:37 02-01-2018] (Error) main.py::confirm.334: Token konnte nicht ausgewertet werden (hhrgswcnm8jy71he014f2xx65xbwzbxa3vdoqx7stgkxxoibva)
[19:52:16 02-01-2018] (Debug) main.py::<module>.1290: Server Reload...
[19:53:51 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:54:15 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:54:18 02-01-2018] (Success) main.py::confirm.359: Benutzer friedrm1@hs-albsig.de wurde bestätigt
[19:54:36 02-01-2018] (Success) main.py::login.179: 
[19:55:15 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[19:55:18 02-01-2018] (Success) main.py::logout.205: Benutzer wurde ausgeloggt: friedrm1@hs-albsig.de
[19:55:28 02-01-2018] (Success) main.py::login.179: Erfolgreicher Login: friedrm1@hs-albsig.de
[20:09:49 02-01-2018] (Debug) main.py::<module>.1285: Server Reload...
[20:10:28 02-01-2018] (Debug) main.py::<module>.1287: Server Reload...
[11:33:03 06-01-2018] (Debug) main.py::<module>.1287: Server Reload...
[11:33:04 06-01-2018] (Debug) main.py::<module>.1287: Server Reload...
[11:38:00 06-01-2018] (Success) main.py::send_verification_email.1227: Bestätigungsemail gesendet an 'marc.friedrich94@web.de' ...
[11:38:00 06-01-2018] (Success) main.py::post_user.274: Registrierung war erfolgreich für Benutzer: marc.friedrich94@web.de. Sende Bestätigungsmail...
[11:38:18 06-01-2018] (Success) main.py::confirm.361: Benutzer marc.friedrich94@web.de wurde bestätigt
[11:38:32 06-01-2018] (Success) main.py::login.181: Erfolgreicher Login: marc.friedrich94@web.de
[11:38:40 06-01-2018] (Success) main.py::logout.207: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[11:39:14 06-01-2018] (Success) main.py::login.181: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[11:39:17 06-01-2018] (Success) main.py::admin_dashboard.314: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[11:39:44 06-01-2018] (Success) main.py::logout.207: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[11:39:53 06-01-2018] (Error) main.py::handle_password_reset.1023: Konnte keinen Benutzer mit der E-Mail marc.friedrich94@webb.de finden.
[11:40:09 06-01-2018] (Success) main.py::handle_password_reset.1044: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc.friedrich94@web.de
[11:41:08 06-01-2018] (Error) main.py::confirm_password_reset.1071: Konnte die übergebenen URL-Parameter nicht auswerten.
[11:41:27 06-01-2018] (Error) main.py::confirm_password_reset.1079: Kein zugehöriger Passwort Reset Request für Token None gefunden.
[11:41:37 06-01-2018] (Error) main.py::confirm_password_reset.1071: Konnte die übergebenen URL-Parameter nicht auswerten.
[11:41:52 06-01-2018] (Error) main.py::confirm_password_reset.1079: Kein zugehöriger Passwort Reset Request für Token None gefunden.
[11:42:44 06-01-2018] (Success) main.py::set_new_password.1142: Passwort für Benutzer 7 wurde geändert.
[11:42:44 06-01-2018] (Debug) main.py::set_new_password.1146: Logge Benutzer 7 aus, um sich mit den geänderten Credentials erneut einzuloggen.
[11:42:56 06-01-2018] (Success) main.py::login.181: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[12:07:22 06-01-2018] (Debug) main.py::<module>.1287: Server Reload...
[12:07:34 06-01-2018] (Error) main.py::post_message.416: Post von Benutzer marc.friedrich94@web.de (meine nachricht...) konnte nicht gepostet werden (Datenbankfehler)
[12:07:47 06-01-2018] (Debug) main.py::<module>.1287: Server Reload...
[12:07:56 06-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[12:08:19 06-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[12:09:34 06-01-2018] (Debug) main.py::<module>.1287: Server Reload...
[12:09:48 06-01-2018] (Debug) main.py::<module>.1286: Server Reload...
[12:10:11 06-01-2018] (Success) main.py::post_message.419: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[12:13:52 06-01-2018] (Debug) main.py::<module>.1290: Server Reload...
[12:14:02 06-01-2018] (Debug) main.py::post_message.401: Bleached message: &lt;script&gt;alert('test');&lt;/script&gt;
[12:14:02 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[12:40:57 06-01-2018] (Success) main.py::vote.478: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[12:41:28 06-01-2018] (Debug) main.py::<module>.1290: Server Reload...
[12:41:32 06-01-2018] (Success) main.py::vote.478: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[12:45:44 06-01-2018] (Debug) main.py::<module>.1298: Server Reload...
[12:45:55 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[13:02:44 06-01-2018] (Error) main.py::check_if_valid_session.1272: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[13:02:51 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[13:02:59 06-01-2018] (Success) main.py::vote.478: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:04:53 06-01-2018] (Debug) main.py::<module>.1304: Server Reload...
[13:04:54 06-01-2018] (Debug) main.py::<module>.1304: Server Reload...
[13:04:56 06-01-2018] (Success) main.py::vote.481: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:09:48 06-01-2018] (Debug) main.py::<module>.1304: Server Reload...
[13:09:49 06-01-2018] (Success) main.py::vote.481: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:13:12 06-01-2018] (Debug) main.py::<module>.1304: Server Reload...
[13:13:13 06-01-2018] (Debug) main.py::<module>.1304: Server Reload...
[13:13:18 06-01-2018] (Success) main.py::vote.481: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:15:07 06-01-2018] (Debug) main.py::<module>.1306: Server Reload...
[13:15:09 06-01-2018] (Error) main.py::vote.458: Ungültiger Zugriff (Post ID oder Methode)
[13:16:10 06-01-2018] (Success) main.py::vote.483: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:16:13 06-01-2018] (Error) main.py::vote.474: Fehlerhafter Link für Vote. Möglicherweise wurde dieser manipuliert.
[13:16:20 06-01-2018] (Success) main.py::vote.483: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:16:40 06-01-2018] (Error) main.py::vote.454: Ungültiger Zugriff (Post ID oder Methode)
[13:22:13 06-01-2018] (Debug) main.py::<module>.1306: Server Reload...
[13:22:13 06-01-2018] (Debug) main.py::<module>.1306: Server Reload...
[13:22:17 06-01-2018] (Error) main.py::vote.454: Ungültiger Zugriff (Post ID oder Methode)
[13:22:51 06-01-2018] (Success) main.py::vote.483: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[13:23:21 06-01-2018] (Error) main.py::finish_vote.521: Vote konnte nicht abgeschlossen werden (keine Übereinstimmung der Token)
[13:23:39 06-01-2018] (Success) main.py::finish_vote.550: Upvote für 6 von marc.friedrich94@web.de wurde registriert.
[13:24:19 06-01-2018] (Success) main.py::vote.466: Benutzer marc.friedrich94@web.de hat bereits für Post 6 abgestimmt
[13:25:03 06-01-2018] (Error) main.py::vote.458: Ungültiger Zugriff (Post ID oder Methode)
[13:25:10 06-01-2018] (Error) main.py::vote.458: Ungültiger Zugriff (Post ID oder Methode)
[13:25:17 06-01-2018] (Error) main.py::vote.458: Ungültiger Zugriff (Post ID oder Methode)
[13:25:23 06-01-2018] (Error) main.py::vote.458: Ungültiger Zugriff (Post ID oder Methode)
[13:25:35 06-01-2018] (Success) main.py::vote.483: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[14:12:30 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[14:12:35 06-01-2018] (Success) main.py::admin_confirm.987: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 1
[14:12:48 06-01-2018] (Error) main.py::check_if_valid_session.1277: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[14:12:55 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[14:12:55 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[14:12:57 06-01-2018] (Success) main.py::admin_confirm.987: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 1
[14:12:58 06-01-2018] (Error) main.py::delete_user.935: Eingegebenes Passwort des Administrators war falsch.
[14:13:00 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[14:13:01 06-01-2018] (Success) main.py::admin_confirm.987: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 1
[14:15:16 06-01-2018] (Success) main.py::delete_user.946: Benutzer 1 wurde gelöscht.
[14:15:19 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[15:30:32 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[15:34:08 06-01-2018] (Error) main.py::check_if_valid_session.1277: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[15:34:14 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[15:34:22 06-01-2018] (Debug) main.py::post_message.401: Bleached message: Text (lateinisch texere weben, flechten) bezeichnet im nichtwissenschaftlichen Sprachgebrauch eine abgegrenzte, zusammenhängende, meist schriftliche sprachliche Äußerung, im weiteren Sinne auch nicht geschriebene, aber schreibbare Sprachinformation (beispielsweise eines Liedes, Films oder einer improvisierten Theater­aufführung). Aus sprachwissenschaftlicher Sicht ist ein Text die sprachliche Form einer kommunikativen Handlung.
[15:34:22 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:35:09 06-01-2018] (Debug) main.py::post_message.401: Bleached message: Text (lateinisch texere weben, flechten) bezeichnet im

nichtwissenschaftlichen Sprachgebrauch eine



abgegrenzte, zusammenhängende, meist schriftliche sprachliche Äußerung, im weiteren Sinne auch nicht geschriebene, aber schreibbare Sprachinformation (beispielsweise eines Liedes, Films oder einer improvisierten Theater­aufführung). Aus sprachwissenschaftlicher Sicht ist ein Text die sprachliche Form einer kommunikativen Handlung.
[15:35:09 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:38:48 06-01-2018] (Debug) main.py::<module>.1306: Server Reload...
[15:42:39 06-01-2018] (Debug) main.py::<module>.1315: Server Reload...
[15:43:10 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:43:44 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:43:55 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:44:16 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:44:52 06-01-2018] (Debug) main.py::<module>.1314: Server Reload...
[15:45:02 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:45:12 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:45:23 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:45:58 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:46:22 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:47:55 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:48:14 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:48:58 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:49:24 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:49:51 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:50:52 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:51:09 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:51:10 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:52:20 06-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[15:53:57 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[15:54:06 06-01-2018] (Debug) main.py::check_if_valid_session.1289: -1
[15:54:06 06-01-2018] (Debug) main.py::post_message.401: Bleached message: test
[15:54:06 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:55:11 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:55:19 06-01-2018] (Debug) main.py::check_if_valid_session.1289: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'WHERE uid = 7' at line 1")
[15:55:19 06-01-2018] (Debug) main.py::post_message.401: Bleached message: test
[15:55:19 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:57:17 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:57:18 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:57:23 06-01-2018] (Debug) main.py::check_if_valid_session.1289: refreshed session for 7 with session_start=2018-01-06 15:57:23.888543 session_max_alive=2018-01-06 16:57:23.888543
[15:57:24 06-01-2018] (Debug) main.py::post_message.401: Bleached message: awdawd
[15:57:24 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:57:56 06-01-2018] (Debug) main.py::check_if_valid_session.1289: refreshed session for 7 with session_start=2018-01-06 15:57:56.953174 session_max_alive=2018-01-06 16:57:56.953174
[15:57:57 06-01-2018] (Debug) main.py::post_message.401: Bleached message: wwww
[15:57:57 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:58:51 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:58:56 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:59:01 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[15:59:20 06-01-2018] (Debug) main.py::check_if_valid_session.1289: 1
[15:59:20 06-01-2018] (Debug) main.py::post_message.401: Bleached message: awdawd
[15:59:20 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[15:59:30 06-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[15:59:40 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[15:59:49 06-01-2018] (Debug) main.py::check_if_valid_session.1289: 1
[15:59:49 06-01-2018] (Debug) main.py::post_message.401: Bleached message: awdawd
[15:59:49 06-01-2018] (Success) main.py::post_message.423: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[16:00:18 06-01-2018] (Debug) main.py::<module>.1316: Server Reload...
[16:00:48 06-01-2018] (Debug) main.py::<module>.1313: Server Reload...
[16:00:56 06-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[16:04:07 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:34:43 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:35:36 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:37:31 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:39:13 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:39:36 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:39:37 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:39:45 06-01-2018] (Error) main.py::check_if_valid_session.1281: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[17:39:52 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[17:39:58 06-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[17:47:01 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:47:14 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:47:26 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:47:32 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[17:48:52 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:49:08 06-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[17:49:21 06-01-2018] (Success) main.py::send_verification_email.1232: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[17:49:21 06-01-2018] (Success) main.py::post_user.275: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[17:54:03 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:54:15 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:58:03 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[17:58:21 06-01-2018] (Error) main.py::post_user.265: Fehler bei Registrierung. Benutzer existiert bereits.
[17:58:48 06-01-2018] (Success) main.py::send_verification_email.1232: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[17:58:48 06-01-2018] (Success) main.py::post_user.275: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[17:59:05 06-01-2018] (Success) main.py::confirm.362: Benutzer friedrm1@hs-albsig.de wurde bestätigt
[18:04:57 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:05:08 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:05:52 06-01-2018] (Success) main.py::send_verification_email.1232: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[18:05:52 06-01-2018] (Success) main.py::post_user.275: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[18:06:49 06-01-2018] (Success) main.py::confirm.362: Benutzer friedrm1@hs-albsig.de wurde bestätigt
[18:09:35 06-01-2018] (Success) main.py::send_verification_email.1232: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[18:09:35 06-01-2018] (Success) main.py::post_user.275: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[18:09:57 06-01-2018] (Success) main.py::confirm.362: Benutzer friedrm1@hs-albsig.de wurde bestätigt
[18:11:23 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:11:38 06-01-2018] (Success) main.py::send_verification_email.1232: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[18:11:38 06-01-2018] (Success) main.py::post_user.275: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[18:11:52 06-01-2018] (Success) main.py::confirm.362: Benutzer friedrm1@hs-albsig.de wurde bestätigt
[18:12:14 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:12:20 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:12:26 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:14:32 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:15:20 06-01-2018] (Error) main.py::post_user.265: Fehler bei Registrierung. Benutzer existiert bereits.
[18:19:07 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:20:52 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:22:08 06-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:22:18 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:22:41 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:22:42 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:22:44 06-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:22:53 06-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:27:03 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:27:11 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:27:48 06-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[18:28:10 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:28:16 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:28:25 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:31:33 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:31:43 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:32:00 06-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:32:04 06-01-2018] (Success) main.py::finish_vote.547: Upvote für 17 von marc.friedrich94@web.de wurde registriert.
[18:36:08 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:37:17 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:37:21 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:37:30 06-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 17 abgestimmt
[18:37:33 06-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:37:36 06-01-2018] (Success) main.py::finish_vote.561: Downvote für 16 von marc.friedrich94@web.de wurde registriert.
[18:37:40 06-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:37:43 06-01-2018] (Success) main.py::finish_vote.547: Upvote für 15 von marc.friedrich94@web.de wurde registriert.
[18:51:59 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:52:02 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:52:49 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:52:52 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:53:15 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:53:20 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:53:36 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:53:37 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:54:13 06-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 17 abgestimmt
[18:54:16 06-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:54:18 06-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 15 abgestimmt
[18:54:20 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[18:54:29 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:54:30 06-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[18:54:32 06-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 17 abgestimmt
[18:54:34 06-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[10:42:14 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[10:42:15 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[10:54:26 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[10:54:30 07-01-2018] (Error) main.py::check_if_valid_session.1281: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[10:54:39 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[10:54:41 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 17 abgestimmt
[10:54:44 07-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[10:54:47 07-01-2018] (Success) main.py::finish_vote.547: Upvote für 13 von marc.friedrich94@web.de wurde registriert.
[10:54:55 07-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[10:54:57 07-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[10:55:00 07-01-2018] (Success) main.py::finish_vote.547: Upvote für 18 von marc.friedrich94@web.de wurde registriert.
[10:55:58 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[10:56:06 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[10:56:24 07-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[10:56:34 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[10:56:35 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[10:56:41 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[10:57:48 07-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[10:57:50 07-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[10:57:54 07-01-2018] (Success) main.py::finish_vote.547: Upvote für 19 von marc.friedrich94@web.de wurde registriert.
[10:57:58 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 19 abgestimmt
[11:01:07 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:01:09 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:01:15 07-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[11:01:22 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[11:01:34 07-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:03:20 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:03:32 07-01-2018] (Error) main.py::check_if_valid_session.1281: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[11:03:40 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[11:03:42 07-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[11:03:49 07-01-2018] (Error) main.py::check_if_valid_session.1281: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[11:04:13 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[11:04:19 07-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[11:06:40 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:06:45 07-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[11:09:39 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:17:47 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:19:19 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:27:55 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:30:12 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:41:50 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:43:17 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:43:20 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:44:57 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:45:00 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:48:17 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:48:22 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:55:42 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:55:45 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:57:58 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:58:02 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:58:13 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:58:49 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:59:39 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[11:59:41 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:00:55 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:00:59 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:04:12 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:04:16 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:04:32 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:04:33 07-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[12:05:17 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[12:05:28 07-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[12:05:41 07-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[12:05:44 07-01-2018] (Success) main.py::finish_vote.547: Upvote für 22 von marc.friedrich94@web.de wurde registriert.
[12:05:48 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 22 abgestimmt
[12:05:50 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 22 abgestimmt
[12:05:53 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 22 abgestimmt
[12:05:55 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 22 abgestimmt
[12:18:59 07-01-2018] (Success) main.py::change_email_handler.650: Bestätigungsmail für E-Mail Änderung wurde an marc.friedrich94@web.de gesendet.
[12:19:09 07-01-2018] (Success) main.py::confirm_email_change.705: Benutzer 7 hat seine E-Mail geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[12:19:09 07-01-2018] (Debug) main.py::confirm_email_change.708: Logge User aus, um sich mit den neuen Credentials anzumelden.
[12:19:20 07-01-2018] (Error) main.py::login.150: Gescheiterter Login (Datenbankfehler): marc.friedrich94@web.de
[12:19:33 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: friedrm1@hs-albsig.de (ADMINISTRATOR)
[12:19:45 07-01-2018] (Success) main.py::change_email_handler.650: Bestätigungsmail für E-Mail Änderung wurde an friedrm1@hs-albsig.de gesendet.
[12:19:58 07-01-2018] (Success) main.py::confirm_email_change.705: Benutzer 7 hat seine E-Mail geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[12:19:58 07-01-2018] (Debug) main.py::confirm_email_change.708: Logge User aus, um sich mit den neuen Credentials anzumelden.
[12:20:08 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[12:20:17 07-01-2018] (Success) main.py::change_password_handler.798: Bestätigungsmail für Passwortänderung wurde an marc.friedrich94@web.de gesendet.
[12:20:27 07-01-2018] (Success) main.py::confirm_password_change.853: Benutzer 7 hat Passwort geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[12:20:27 07-01-2018] (Debug) main.py::confirm_password_change.856: Logge User aus, um sich mit den neuen Credentials anzumelden.
[12:20:40 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[12:20:42 07-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[12:20:48 07-01-2018] (Success) main.py::handle_password_reset.1049: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc.friedrich94@web.de
[12:21:16 07-01-2018] (Success) main.py::set_new_password.1147: Passwort für Benutzer 7 wurde geändert.
[12:21:16 07-01-2018] (Debug) main.py::set_new_password.1151: Logge Benutzer 7 aus, um sich mit den geänderten Credentials erneut einzuloggen.
[12:21:22 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[12:21:24 07-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[12:21:27 07-01-2018] (Success) main.py::admin_confirm.984: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 6
[12:21:29 07-01-2018] (Error) main.py::delete_user.939: Benutzer 6 konnte nicht gelöscht werden (Datenbankfehler). Stacktrace: (1305, 'PROCEDURE tralala.delete_user does not exist')
[12:22:02 07-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[12:22:28 07-01-2018] (Success) main.py::admin_confirm.984: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 6
[12:22:31 07-01-2018] (Success) main.py::delete_user.943: Benutzer 6 wurde gelöscht.
[12:22:32 07-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[13:27:10 07-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[13:27:39 07-01-2018] (Success) main.py::send_verification_email.1232: Bestätigungsemail gesendet an 'friedrm1@hs-albsig.de' ...
[13:27:39 07-01-2018] (Success) main.py::post_user.275: Registrierung war erfolgreich für Benutzer: friedrm1@hs-albsig.de. Sende Bestätigungsmail...
[13:28:56 07-01-2018] (Success) main.py::confirm.362: Benutzer friedrm1@hs-albsig.de wurde bestätigt
[16:01:52 07-01-2018] (Error) main.py::post_user.265: Fehler bei Registrierung. Benutzer existiert bereits.
[16:05:41 07-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[16:05:55 07-01-2018] (Success) main.py::post_message.420: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[16:05:58 07-01-2018] (Success) main.py::vote.480: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[16:06:01 07-01-2018] (Success) main.py::finish_vote.547: Upvote für 23 von marc.friedrich94@web.de wurde registriert.
[16:06:04 07-01-2018] (Success) main.py::vote.463: Benutzer marc.friedrich94@web.de hat bereits für Post 23 abgestimmt
[15:31:00 09-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:31:00 09-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:31:15 09-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[15:31:18 09-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[15:31:30 09-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:42:33 09-01-2018] (Debug) main.py::<module>.1312: Server Reload...
[15:43:23 09-01-2018] (Debug) main.py::<module>.1315: Server Reload...
[15:43:23 09-01-2018] (Debug) main.py::<module>.1315: Server Reload...
[15:44:39 09-01-2018] (Debug) main.py::<module>.1315: Server Reload...
[15:45:11 09-01-2018] (Debug) main.py::<module>.1315: Server Reload...
[15:45:51 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:47:05 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:47:11 09-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[15:47:21 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:47:40 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:47:44 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:47:52 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:48:32 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:48:48 09-01-2018] (Debug) main.py::<module>.1318: Server Reload...
[15:49:10 09-01-2018] (Success) main.py::admin_dashboard.315: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[15:49:20 09-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[15:51:52 09-01-2018] (Debug) main.py::<module>.1322: Server Reload...
[15:52:03 09-01-2018] (Error) main.py::admin_dashboard.294: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[15:52:15 09-01-2018] (Error) main.py::admin_dashboard.294: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[15:52:16 09-01-2018] (Error) main.py::admin_dashboard.294: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[15:52:18 09-01-2018] (Error) main.py::admin_dashboard.294: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[15:52:19 09-01-2018] (Error) main.py::admin_dashboard.294: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[15:52:31 09-01-2018] (Success) main.py::login.182: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[15:52:38 09-01-2018] (Success) main.py::admin_dashboard.319: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[15:52:49 09-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[15:52:58 09-01-2018] (Success) main.py::login.182: Erfolgreicher Login: mueller.dieter@hans-maier.de
[15:53:05 09-01-2018] (Error) main.py::admin_dashboard.304: Unbefugter Benutzer 'mueller.dieter@hans-maier.de versucht auf das Admin Dashboard zuzugreifen. Verweigere Zugriff.
[15:53:11 09-01-2018] (Success) main.py::logout.208: Benutzer wurde ausgeloggt: mueller.dieter@hans-maier.de
[15:53:29 09-01-2018] (Debug) main.py::<module>.1321: Server Reload...
[15:54:03 09-01-2018] (Debug) main.py::<module>.1324: Server Reload...
[15:54:28 09-01-2018] (Debug) main.py::<module>.1324: Server Reload...
[15:54:51 09-01-2018] (Debug) main.py::<module>.1327: Server Reload...
[15:54:56 09-01-2018] (Error) main.py::admin_dashboard.293: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[15:55:15 09-01-2018] (Error) main.py::admin_confirm.967: Ein nicht eingeloggter Benutzer wollte /auth/admin/confirm aufrufen.
[16:00:25 09-01-2018] (Error) main.py::admin_dashboard.293: Ein nicht eingeloggter Benutzer wollte auf das Admin Dashboard zugreifen.
[20:24:06 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:24:06 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:24:23 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[20:24:36 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[20:25:24 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:25:25 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:26:33 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:26:34 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:31:41 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:31:42 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:33:05 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:33:06 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:33:21 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:33:21 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:33:37 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[20:35:22 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:35:22 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:36:03 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:36:03 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:36:18 09-01-2018] (Success) main.py::login.200: Erfolgreicher Login: Admin.admin@hans-maier.de
[20:37:07 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:37:07 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:37:15 09-01-2018] (Success) main.py::logout.226: Benutzer wurde ausgeloggt: Admin.admin@hans-maier.de
[20:37:20 09-01-2018] (Success) main.py::login.200: Erfolgreicher Login: Admin.admin@hans-maier.de
[20:37:34 09-01-2018] (Success) main.py::logout.226: Benutzer wurde ausgeloggt: Admin.admin@hans-maier.de
[20:37:43 09-01-2018] (Success) main.py::login.200: Erfolgreicher Login: Admin.admin@hans-maier.de
[20:38:24 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:38:25 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:38:28 09-01-2018] (Success) main.py::logout.226: Benutzer wurde ausgeloggt: Admin.admin@hans-maier.de
[20:39:15 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:39:16 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:39:26 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[20:40:44 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:40:45 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:43:59 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:43:59 09-01-2018] (Debug) main.py::<module>.1345: Server Reload...
[20:44:15 09-01-2018] (Error) main.py::login.158: Account loggedAdmin.admin@hans-maier.de
[21:26:37 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:26:37 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:26:51 09-01-2018] (Error) main.py::login.158: Account loggedAdmin.admin@hans-maier.de
[21:28:06 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:28:06 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:28:13 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[21:28:20 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[21:28:24 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[21:28:29 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[21:29:49 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[21:30:01 09-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): Admin.admin@hans-maier.de
[21:32:03 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:32:04 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:33:25 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:33:26 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:33:35 09-01-2018] (Success) main.py::login.202: Erfolgreicher Login: Admin.admin@hans-maier.de
[21:33:39 09-01-2018] (Success) main.py::logout.228: Benutzer wurde ausgeloggt: Admin.admin@hans-maier.de
[21:34:20 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:34:20 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:34:26 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: -1
[21:34:29 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: -1
[21:35:07 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:35:08 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:36:51 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:36:52 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:37:03 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: 0
[21:38:23 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:38:33 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: -1
[21:38:37 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: -1
[21:39:19 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:39:20 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:39:27 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: 0
[21:39:34 09-01-2018] (Error) main.py::login.158: Account loggedAdmin.admin@hans-maier.de
[21:40:46 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:43:00 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:43:17 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:46:57 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:46:57 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:47:12 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: 0
[21:47:39 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: -1
[21:48:12 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:48:13 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[21:48:19 09-01-2018] (Error) main.py::login.179: Gescheiterter Login (Übereinstimmung): Admin.admin@hans-maier.de	Lockedcounter return: 0
[22:19:43 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:19:43 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:19:54 09-01-2018] (Success) main.py::login.202: Erfolgreicher Login: Admin.admin@hans-maier.de
[22:20:14 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:21:33 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:21:48 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:23:13 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:23:14 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:26:15 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:27:38 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:27:38 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:27:50 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:28:15 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:28:16 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:29:00 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:29:00 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:29:08 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:29:09 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:30:22 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:30:22 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:30:30 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:33:51 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:33:52 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:34:04 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:36:06 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:36:07 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:37:06 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:37:07 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:37:19 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:37:44 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:37:44 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:37:44 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:37:53 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:38:42 09-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[22:39:13 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[22:39:13 09-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[00:17:52 10-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[00:17:53 10-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[00:21:00 10-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[00:21:01 10-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[00:22:46 10-01-2018] (Success) main.py::logout.228: Benutzer wurde ausgeloggt: Admin.admin@hans-maier.de
[00:22:56 10-01-2018] (Success) main.py::login.202: Erfolgreicher Login: Admin.admin@hans-maier.de (ADMINISTRATOR)
[00:22:59 10-01-2018] (Success) main.py::admin_dashboard.338: Administrator 'Admin.admin@hans-maier.de hat Admin Dashboard geladen
[00:25:02 10-01-2018] (Success) main.py::logout.228: Benutzer wurde ausgeloggt: Admin.admin@hans-maier.de
[14:30:58 17-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[14:30:58 17-01-2018] (Debug) main.py::<module>.1347: Server Reload...
[14:31:09 17-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): marc.friedrich94@web.de
[14:32:00 17-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): marc.friedrich94@web.de
[14:33:05 17-01-2018] (Success) main.py::send_verification_email.1261: Bestätigungsemail gesendet an 'marc.friedrich94@web.de' ...
[14:33:05 17-01-2018] (Success) main.py::post_user.295: Registrierung war erfolgreich für Benutzer: marc.friedrich94@web.de. Sende Bestätigungsmail...
[14:33:22 17-01-2018] (Success) main.py::confirm.385: Benutzer marc.friedrich94@web.de wurde bestätigt
[14:33:30 17-01-2018] (Error) main.py::login.164: Gescheiterter Login (Datenbankfehler): marc.friedrich94@web.de
[14:36:51 17-01-2018] (Success) main.py::send_verification_email.1261: Bestätigungsemail gesendet an 'marc.friedrich94@web.de' ...
[14:36:51 17-01-2018] (Success) main.py::post_user.295: Registrierung war erfolgreich für Benutzer: marc.friedrich94@web.de. Sende Bestätigungsmail...
[14:37:08 17-01-2018] (Success) main.py::confirm.385: Benutzer marc.friedrich94@web.de wurde bestätigt
[14:37:15 17-01-2018] (Success) main.py::login.202: Erfolgreicher Login: marc.friedrich94@web.de
[14:37:24 17-01-2018] (Success) main.py::vote.503: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[14:37:26 17-01-2018] (Success) main.py::finish_vote.570: Upvote für 2 von marc.friedrich94@web.de wurde registriert.
[14:37:29 17-01-2018] (Success) main.py::vote.486: Benutzer marc.friedrich94@web.de hat bereits für Post 2 abgestimmt
[14:46:30 17-01-2018] (Debug) main.py::<module>.1351: Server Reload...
[14:46:30 17-01-2018] (Debug) main.py::<module>.1351: Server Reload...
[14:47:56 17-01-2018] (Debug) main.py::<module>.1352: Server Reload...
[14:48:10 17-01-2018] (Debug) main.py::<module>.1352: Server Reload...
[14:48:30 17-01-2018] (Debug) main.py::<module>.1352: Server Reload...
[14:49:26 17-01-2018] (Debug) main.py::<module>.1352: Server Reload...
[14:51:29 17-01-2018] (Debug) main.py::<module>.1357: Server Reload...
[14:51:58 17-01-2018] (Debug) main.py::<module>.1358: Server Reload...
[14:53:21 17-01-2018] (Debug) main.py::<module>.1360: Server Reload...
[16:01:50 17-01-2018] (Error) main.py::check_if_valid_session.1323: Session Timeout wurde erreicht. Automatischer Logout für Benutzer marc.friedrich94@web.de
[16:02:02 17-01-2018] (Success) main.py::login.202: Erfolgreicher Login: marc.friedrich94@web.de
[16:02:07 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[16:02:25 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[16:03:16 17-01-2018] (Debug) main.py::<module>.1360: Server Reload...
[16:04:06 17-01-2018] (Debug) main.py::<module>.1360: Server Reload...
[16:04:26 17-01-2018] (Debug) main.py::<module>.1362: Server Reload...
[16:04:41 17-01-2018] (Debug) main.py::<module>.1362: Server Reload...
[16:10:44 17-01-2018] (Debug) main.py::<module>.1362: Server Reload...
[16:11:02 17-01-2018] (Debug) main.py::<module>.1362: Server Reload...
[16:11:26 17-01-2018] (Debug) main.py::<module>.1362: Server Reload...
[16:12:54 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:13:00 17-01-2018] (Debug) main.py::<module>.1367: Server Reload...
[16:13:24 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:13:52 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:14:10 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:14:14 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:14:25 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:14:32 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:15:05 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:15:09 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:15:23 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:16:28 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:18:16 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:19:11 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:19:25 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:19:27 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:19:44 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:23:37 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:23:41 17-01-2018] (Debug) main.py::<module>.1365: Server Reload...
[16:25:07 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:25:08 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:25:51 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:25:54 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:26:06 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:27:15 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:48:15 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:54:02 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[16:55:29 17-01-2018] (Success) main.py::vote.486: Benutzer marc.friedrich94@web.de hat bereits für Post 2 abgestimmt
[16:55:31 17-01-2018] (Success) main.py::vote.503: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[16:55:34 17-01-2018] (Success) main.py::finish_vote.570: Upvote für 1 von marc.friedrich94@web.de wurde registriert.
[16:56:49 17-01-2018] (Success) main.py::vote.486: Benutzer marc.friedrich94@web.de hat bereits für Post 2 abgestimmt
[17:01:36 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[17:03:05 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[17:03:44 17-01-2018] (Debug) main.py::<module>.1371: Server Reload...
[17:04:54 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[17:05:01 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[17:05:05 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[17:05:10 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[17:05:17 17-01-2018] (Success) main.py::post_message.443: Neue Nachricht von Benutzer marc.friedrich94@web.de geposted
[17:09:08 17-01-2018] (Success) main.py::vote.503: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[17:09:11 17-01-2018] (Success) main.py::finish_vote.570: Upvote für 8 von marc.friedrich94@web.de wurde registriert.
[19:04:35 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:04:35 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:07:26 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:08:33 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:08:33 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:14:27 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:14:27 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:16:28 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:16:32 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:16:32 19-01-2018] (Debug) main.py::<module>.1331: Server Reload...
[19:16:48 19-01-2018] (Success) main.py::login.216: Erfolgreicher Login: Admin.admin@hans-maier.de (ADMINISTRATOR)
[19:16:55 19-01-2018] (Success) main.py::post_message.456: Neue Nachricht von Benutzer Admin.admin@hans-maier.de geposted
[19:53:36 19-01-2018] (Debug) main.py::<module>.1525: Server Reload...
